CMakeLists.txt

```diff
@@ -39,6 +44,16 @@ endif()
 add_library(docopt SHARED ${DOCOPT_SRC})
 target_include_directories(docopt PUBLIC "${PROJECT_SOURCE_DIR}")
 
+# Use BOOST for GCC 4.8
+if(CMAKE_COMPILER_IS_GNUCC AND ${CMAKE_CXX_COMPILER_VERSION} VERSION_LESS 4.9)
+    find_package(Boost REQUIRED COMPONENTS Regex)
+    include_directories(${Boost_INCLUDE_DIRS})
+    target_link_libraries(docopt ${Boost_LIBRARIES})
+	if(WITH_STATIC)
+		target_link_libraries(docopt_s ${Boost_LIBRARIES})
+	endif()
+endif()
+
```

Comment:
I forgot to update this part. The condition in the if should be `if(USE_BOOST_REGEX)`, and maybe add an error message with a suggestion to define this variable.